vnext/Microsoft.ReactNative/Modules/AccessibilityInfoModule.cpp

```diff
@@ -50,6 +52,7 @@ void AccessibilityInfo::setAccessibilityFocus(double /*reactTag*/) noexcept {
 
 void AccessibilityInfo::announceForAccessibility(std::wstring announcement) noexcept {
```

Comment:
Is there an issue tracking that this API doesn't work in fabric?

Reply:
There is now: https://github.com/microsoft/react-native-windows/issues/14296